misc/docker/alpine/3.13/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM alpine:3.13
+ARG test
 RUN apk update
 RUN apk add build-base linux-headers git bash libunwind-dev
 RUN mkdir -p /usr/src
 RUN git clone https://github.com/namhyung/uftrace /usr/src/uftrace
-RUN cd /usr/src/uftrace && ./misc/install-deps.sh && ./configure && make && make install
+RUN if [ "${test}" == "yes" ] ; then \
+        cd /usr/src/uftrace && ./misc/install-deps.sh && ./configure && make && make unittest; \
```

Comment:
i didn't add ASAN=1 in alpine because AddressSanitizer [doesn't support  musl](https://github.com/google/sanitizers/issues/1080)